lib/commands/push.js

```diff
@@ -1672,7 +1677,150 @@ const pushFunction = async ({ functionId, async, code, withVariables } = { retur
+            });
+        }
+    }));
+
+
+    if (!(await approveChanges(tables, gridsGetTable, KeysTable, 'tableId', 'tables', ['columns', 'indexes'], 'databaseId', 'databaseId',))) {
+        return;
+    }
+    // Parallel collection actions
+    await Promise.all(tables.map(async (table) => {
+        try {
+            const remoteTable = await gridsGetTable({
+                databaseId: table['databaseId'],
+                tableId: table['$id'],
+                parseOutput: false,
+            });
+
+            if (remoteTable.name !== table.name) {
+                await databasesUpdateTable({
+                    databaseId: table['databaseId'],
+                    tableId: table['$id'],
+                    name: table.name,
+                    name: table.name,
+                    parseOutput: false
+                })
+
+                success(`Updated ${table.name} ( ${table['$id']} ) name`);
+            }
+            table.remoteVersion = remoteTable;
+
+            table.isExisted = true;
+        } catch
+            (e) {
+            if (Number(e.code) === 404) {
+                log(`Table ${table.name} does not exist in the project. Creating ... `);
+                await databasesCreateTable({
+                    databaseId: table['databaseId'],
+                    tableId: table['$id'],
+                    name: table.name,
+                    documentSecurity: table.documentSecurity,
+                    permissions: table['$permissions'],
+                    parseOutput: false
+                })
+            } else {
+                throw e;
+            }
+        }
+    }))
+    let numberOfTables = 0;
+    // Serialize attribute actions
+    for (let table of tables) {
+        let columns = table.columns;
+        let indexes = table.indexes;
+
+        if (table.isExisted) {
+            columns = await attributesToCreate(table.remoteVersion.columns, table.columns, table);
+            indexes = await attributesToCreate(table.remoteVersion.indexes, table.indexes, table, true);
+
+            if ((Array.isArray(columns) && columns.length <= 0) && (Array.isArray(indexes) && indexes.length <= 0)) {
+                continue;
+            }
+
+        }
+
+        log(`Pushing table ${table.name} ( ${table['databaseId']} - ${table['$id']} ) attributes`)
+
+        try {
+            await createAttributes(columns, table)
+        } catch (e) {
+            throw e;
+        }
+
+        try {
+            await createIndexes(indexes, table);
+        } catch (e) {
+            throw e;
+        }
+        numberOfTables++;
+        success(`Successfully pushed ${table.name} ( ${table['$id']} )`);
+    }
+
+    success(`Successfully pushed ${numberOfTables} tables`);
+}
+
 const pushCollection = async ({ returnOnZero, attempts } = { returnOnZero: false }) => {
+    warn("appwrite push collection has been deprecated. Please consider using 'appwrite push tables' instead");
```

Comment:
Is this accurate? you cant just use push tables. you need to delete your database and switch to tables database. If it's this complex, we should suggest it as 1 simple command, instead, recommend to consider switchign to Tables database, for benefits X Y and Z.

Reply:
same in other commends where we have similar warning

Reply:
You don't need to swap anything on backend, just the command. TablesDB is 100% compatible with the legacy API if you use the new terminology